product-mini/platforms/linux-sgx/CMakeLists.txt

```diff
@@ -126,31 +126,27 @@ endif()
 
 if (WAMR_BUILD_LIB_RATS EQUAL 1)
     execute_process(
-        COMMAND bash -c "sed -i -E 's/^#define LIB_RATS 0 /#define LIB_RATS 1/g' ${CMAKE_CURRENT_SOURCE_DIR}/enclave-sample/Enclave/Enclave.edl"
+        COMMAND bash -c "sed -i -E 's/^#define WAMR_ENABLE_LIB_RATS 0/#define WAMR_ENABLE_LIB_RATS 1/g' ${CMAKE_CURRENT_SOURCE_DIR}/enclave-sample/Enclave/Enclave.edl"
```

Comment:
We usally use `WAMR_BUILD_XXX` as cmake variable for make files and `WASM_ENABLE_XXX` as macro for C/C++ source code. So here had better:

`WAMR_ENABLE_LIB_RATS`  -> `WASM_ENABLE_LIB_RATS`